rest-service/manager_rest/rest/resources_v3_1/plugins.py

```diff
@@ -161,18 +162,80 @@ def get(self,
 
 
 class PluginsId(resources_v2_1.PluginsId):
+    @authorize('plugin_upload')
+    @rest_decorators.marshal_with(models.Plugin)
+    def post(self, plugin_id, **kwargs):
+        """Force plugin installation on the given managers or agents.
+
+        This method is for internal use only.
```

Comment:
Is this really for internal use? I thought it was going to be called by users to cause the plugins to be pre-warmed?

Reply:
internal use, as in only to be called by our code, and not documented as a rest endpoint. I don't want to make this public yet, I'm not absolutely sure this is as good as it can be (wrt. POST/PUT). But I don't have a better idea yet. Maybe we can make this public in 5.2.

Reply:
<thumbs up>